lib/simp/cli/config/items/action/set_grub_password_action.rb

```diff
@@ -37,7 +37,15 @@ def apply_summary
     end
 
     def set_password_grub(grub_hash)
-      result = execute("sed -i 's/password_pbkdf2 root.*$/password_pbkdf2 root #{grub_hash}/' /etc/grub.d/01_users")
+      grub_users = '/etc/grub.d/01_users'
```

Comment:
Could we run `#{@puppet_apply_cmd} -e 'grub_user{...}'` instead of maintaining custom grub2 brain-surgery code in simp-cli?